ratatui-widgets/README.md

```diff
@@ -71,6 +75,6 @@ details on contributing, please see the [CONTRIBUTING](CONTRIBUTING.md) document
 
 ## License
 
-This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for details.
+This project is licensed under the MIT License. See the [LICENSE](../LICENSE) file for details.
```

Comment:
I'm curious how this would render in the crates page once packaged, it's likely no big deal, but any idea? Not sure if there's any easy way to test that either.

Reply:
Good question. We should probably use an absolute URL in there. (i.e. `https://github.com/ratatui/ratatui/blob/main/LICENSE`) (I ran `cargo package` to analyze the generated assets just to sanity check that.)